src/services/ai-server/ai-persona-engine-adapter/ai.persona.engine.adapter.ts

```
@@ -60,6 +65,21 @@ export class AiPersonaEngineAdapter {
+          const responseGeneric = this.virtualContributorEngineGeneric.send<
+            AiPersonaEngineAdapterQueryResponse,
+            AiPersonaEngineAdapterQueryInput
+          >({ cmd: AiPersonaEngineEventType.QUERY }, eventData);
+          responseData = await firstValueFrom(responseGeneric);
+          break;
+        case AiPersonaEngine.OPENAI_ASSISTANT:
+          const repsonseOpenaiAssistant =
+            this.virtualContributorEngineOpenaiAssistant.send<
+              AiPersonaEngineAdapterQueryResponse,
+              AiPersonaEngineAdapterQueryInput
+            >({ cmd: AiPersonaEngineEventType.QUERY }, eventData);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)